### ISSUE TITLE
A11y_SQL Azure Data Studio_Notebooks - Code cell_Screenreader : The screenreader does not convey the state of "Trusted"  button that it is activated or not activated.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/azuredatastudio#15862]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#NVDA;#JAWS;#STP

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Win10
Screen reader : JAWS 2021.2105.53 , NVDA2020.2

### Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Press context menu key on connection and hit enter on New Notebook
4. Hit enter on cell and select code cell.
5. Run the screen reader and navigate to the "Trusted" button.
6. Check its operation and state by pressing enter key.

### Actual:
The screen reader does not announce the state of the "Trusted" button that it is activated or not activated.

This issue is observed with both NVDA and JAWS screen readers.

### Expected:
The screen reader must convey the state of the button when it get activated or not activated.

### User Impact:
Screen reader only user will face difficulty to understand and differentiate the functionality of the control.

**Comment by chlafreniere** [comment]
@Ashutoshsahoo164 the screen reader will read either "Trusted" or "Not trusted" depending on whether the button is pressed or not. Is there anything further required here?

**Comment by Ashutoshsahoo164** [comment]
@chlafreniere No , nothing more is required further.

**Comment by Harika1702** [comment]
GitHub Tags:#Closed;
